README.md

```diff
@@ -1 +1,47 @@
-# docker-terraform
+# docker-terraform [![CI](https://github.com/d3b-center/docker-terraform/workflows/CI/badge.svg?branch=master)](https://github.com/d3b-center/docker-terraform/actions?query=workflow%3ACI)
+
+This repository contains a collection of Nix flakes that produce container images designed to run deployments using the AWS CLI and Terraform.
+
+## Usage
+
+Via Docker:
```

Comment:
Per our conversation, please remove this from README to reflect we only intend to use this docker container with docker-compose.yml.